prow/jobs/jobs.yaml

```diff
@@ -3285,7 +3285,7 @@ postsubmits:
           - name: GITHUB_TOKEN
             valueFrom:
               secretKeyRef:
-                name: prowjob-github-token
+                name: prowjob-github-pat-token
```

Comment:
Not sure if this is correct, @jljaco @RedbackThomson ?

Reply:
I believe this is correct

https://github.com/aws-controllers-k8s/test-infra/blob/9b24d142619fd2058415065b7e3ec3a11900b892/infra/lib/charts/prow-secrets.ts#L51